### ISSUE TITLE
Samples Outdated

### ISSUE THREAD

**Issue by landonseey** [SteamRE/SteamKit#261]
I tried multiple samples and none work. It says connecting to steam and always follows with disconnected. Yet, I am connected to Steam on my main account as I test and this bot cannot connect.

**Comment by Netshroud** [comment]
This is likely a duplicate of #229 and #236.

If Sample 7 (ServerList) doesn't work, I'll be quite surprised. Can you try that?

**Comment by landonseey** [comment]
#236 helped. The samples should be updated with SteamDirectory.Initialize().Wait();     **Thanks!**

**Comment by JustArchi** [comment]
It's crucial to understand that you should call that only once per program startup, and if you start the program frequently, you should save and load CMList from persistent storage.

**Comment by JustArchi** [comment]
BTW, it'd be really cool if you could either delete or update built-in list in SK2, as it only causes confusion and waste of time for all newbies wondering what they did wrong, like in issue here.

**Comment by VoiDeD** [comment]
It was mentioned on IRC that we'd be going with updating the built-in list and pushing out a new minor release for the time being. I don't think anyone got around to doing that, however. I'm personally pretty strapped for time due to my job and I think @Netshroud is in the same situation.

@JustArchi, a PR to update the list with the results of the ISteamDirectory call with a cell id of 0 would be a good start, if you're willing to go through the process.

**Comment by JustArchi** [comment]
If I'm capable of not destroying anything, I'm more than happy to help.
